### PR TITLE
Create LICENSE

### PR DESCRIPTION
**Pull request by DRSchlaubi**

### CODE REVIEW

**Review by yk-sgr — CHANGES_REQUESTED** [review]
Replace placeholder text

**Comment by DRSchlaubi** [comment]
> Replace placeholder text

Already did

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+  If you develop a new program, and you want it to be of the greatest
+possible use to the public, the best way to achieve this is to make it
+free software which everyone can redistribute and change under these terms.
+
+  To do so, attach the following notices to the program.  It is safest
+to attach them to the start of each source file to most effectively
+state the exclusion of warranty; and each file should have at least
+the "copyright" line and a pointer to where the full notice is found.
+
+    <one line to give the program's name and a brief idea of what it does.>
```
Comment (yk-sgr) [inline LICENSE]: >   <one line to give the program's name and a brief idea of what it does.>

**Review by yk-sgr — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (DRSchlaubi) [reply LICENSE]: GNU is making this to complicated for my brain

**Review by yk-sgr — APPROVED** [review]

**Pull request merged**